src/save/__tests__/encodeBase64.test.ts

```diff
@@ -0,0 +1,85 @@
+import { createRgbaImage } from '../../../test/testUtils.js';
+import { encode } from '../encode.js';
+import { encodeBase64 } from '../encodeBase64.js';
+
+test('basic image (png)', () => {
+  const image = testUtils.createGreyImage([
+    [0, 0, 0, 0, 0],
+    [0, 255, 255, 255, 0],
+    [0, 255, 0, 255, 0],
+    [0, 255, 255, 255, 0],
+    [255, 0, 255, 0, 255],
+  ]);
+  const base64Url = encodeBase64(image, 'png');
+
+  expect(base64Url).toBe(
+    'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAAAAACoBHk5AAAAFklEQVR4XmNggID///+DSCCEskHM/wCAnQr2TY5mOAAAAABJRU5ErkJggg==',
+  );
+});
+
+test('basic image 2 (jpeg)', () => {
+  const image = testUtils.createGreyImage([
+    [255, 255, 255, 255, 255],
+    [255, 0, 0, 0, 255],
+    [255, 0, 0, 0, 255],
+    [255, 0, 0, 0, 255],
+    [255, 255, 255, 255, 255],
+  ]);
+  const format = 'jpeg';
+  const base64 = encodeBase64(image, format);
+  const base64Data = Buffer.from(encode(image, { format })).toString('base64');
+  expect(typeof base64).toBe('string');
+  expect(base64Data).toMatchSnapshot();
+});
+
+test('legacy image-js test', () => {
+  const image = createRgbaImage([
+    [
+      255,
+      0,
+      0,
+      255, // red
```

Comment:
Can you use a template string variant? It will be more compact / readable. Example: https://github.com/image-js/image-js-typescript/blob/bd5cc06ea5e2e284a7531011d85efefc0635b8f9/test/__tests__/testUtils.test.ts#L129-L132